src/components/CollapsableErrorList.js

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react';
+import { injectIntl } from 'react-intl';
+import ExpandLess from '@material-ui/icons/ExpandLess';
+import ExpandMore from '@material-ui/icons/ExpandMore';
+import {
+  formatMessage,
+} from '@openimis/fe-core';
+import {
+  ListItem,
+  ListItemText,
+  Collapse,
+} from '@material-ui/core';
+import { withTheme, withStyles } from '@material-ui/core/styles';
+
+const styles = (theme) => ({
+  item: theme.paper.item,
+});
+
+function CollapsableErrorList({
+  intl,
+  errors,
+}) {
+  const [isOpen, setIsOpen] = useState(false);
```

Comment:
Maybe it could be better to name this variable 'isExpanded' or 'isCollapsed'

Reply:
changed